include/argparse/argparse.hpp

```diff
@@ -180,9 +181,10 @@ constexpr bool starts_with(std::basic_string_view<CharT, Traits> prefix,
 }
 
 enum class chars_format {
-  scientific = 0x1,
-  fixed = 0x2,
-  hex = 0x4,
+  scientific = 0xf1,
+  fixed = 0xf2,
+  hex = 0xf4,
+  bin = 0xf8,
```

Comment:
Why did you change the one-hot encoding to set all the upper bits?

Reply:
because radix_2 = 2 collides with fixed in template <class T> struct parse_number<T, chars_format::fixed>